mediainfo@aclon/files/mediainfo@aclon/mi.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+P=$1
+cd $P
+shift
+F=$(printf '%q ' "$@")
+act=$(dirname $0)
+csv="$act/po/$(echo $LANG | sed 's/.UTF-8//').csv"
+editor="(vim -c 'set filetype=config' -c 'set ignorecase' || less)"
+format="sed 's/ *:/齾:/' | column -t -L -s '齾' -o ''"
+# format="format"                               # Shorter 1st col mode
+cmd="[ -f $csv ] || $act/wget_lang.sh ; mediainfo --Language=file://$csv $F | $format | $editor ; mediainfo $F | $format | $editor"
+# cmd="bash -c \" . $act/func.sh ; $cmd \""     # Shorter 1st col mode
+x-terminal-emulator -e "$cmd"
```

Comment:
use `x-terminal-emulator` instead of `gnome-terminal` for other DE users. But this is only tested in cinnamon env.

Reply:
Just a heads up, this will only work for Debian-based systems. Outside of that, `x-terminal-emulator` doesn't exist.